src/DAL/migration/fullSchema.sql

```diff
@@ -30,31 +30,30 @@ CREATE TABLE records
     update_date timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
     source_start_date timestamp with time zone,
     source_end_date timestamp with time zone,
-    max_resolution_deg numeric NOT NULL,
-    min_horizontal_accuracy_ce_90 real,
+    max_resolution_deg numeric NOT NULL CHECK (max_resolution_deg BETWEEN 0.00000009 AND 0.072),
+    min_horizontal_accuracy_ce_90 real CHECK (min_horizontal_accuracy_ce_90 BETWEEN 0.01 AND 4000),
```

Comment:
REAL and DOUBLE PRECISION data types can store rational numbers. However, compared to NUMERIC and DECIMAL data types, REAL and DOUBLE PRECISION can store even more digits. The REAL data type has a storage size of 4 bytes and a minimum precision of 6 decimal digits.